crates/sargon/Cargo.toml

```diff
@@ -74,22 +74,22 @@ strum = { git = "https://github.com/Peternator7/strum/", rev = "f746c3699acf1501
   "derive",
 ] }
 
-sbor = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "cuttlefish-2ea8b582" }
-radix-rust = { git = "https://github.com/radixdlt/radixdlt-scrypto", tag = "cuttlefish-2ea8b582", features = [
+sbor = { git = "https://github.com/radixdlt/radixdlt-scrypto", rev = "27da7a08fac63d36bd1ab209840fb014f470ebc6" }
```

Comment:
Can we have constants in Cargo.toml? It would be really nice to have to change this value only once. Not blocking this PR, just wondering

Reply:
Nope not possible I think. I've been wanting it for a loooong time.